tools/shared.py

```diff
@@ -1063,22 +1066,18 @@ def include_directive(paths):
     return result
 
   # libcxx include paths must be defined before libc's include paths otherwise libcxx will not build
-  EMSDK_OPTS = C_OPTS + include_directive(CXX_INCLUDE_PATHS) + include_directive(C_INCLUDE_PATHS)
+  return C_OPTS + include_directive(CXX_INCLUDE_PATHS) + include_directive(C_INCLUDE_PATHS)
 
-  COMPILER_OPTS += EMSDK_OPTS
-else:
-  EMSDK_OPTS = []
 
-# Engine tweaks
+EMSDK_OPTS = emsdk_opts()
+COMPILER_OPTS += EMSDK_OPTS
 
-try:
```

Comment:
This is NFC because SPIDERMONKEY_ENGINE was changed to be always defined right? Nice

Reply:
Right